cmd/org/clusters.go

```diff
@@ -20,39 +18,55 @@ const (
 )
 
 var (
-	clustersCmd = &cobra.Command{
-		Use:           "clusters",
-		Short:         "get organization clusters",
-		Args:          cobra.ArbitraryArgs,
+	allClustersFlag = false
+	awsAccountID    = ""
+	clustersCmd     = &cobra.Command{
+		Use:   "clusters",
+		Short: "get all active organization clusters",
+		Long: `By default, returns all active clusters for a given organization. The organization can either be specified with an argument
+passed in, or by providing both the --aws-profile and --aws-account-id flags. You can request all clusters regardless of status by providing the --all flag.`,
+		Example: `Retrieving all active clusters for a given organizational unit:
+osdctl org clusters 123456789AbcDEfGHiJklMnopQR
+
+Retrieving all active clusters for a given organizational unit in JSON format:
+osdctl org clusters 123456789AbcDEfGHiJklMnopQR -o json
+
+Retrieving all clusters for a given organizational unit regardless of status:
+osdctl org clusters 123456789AbcDEfGHiJklMnopQR --all
+
+Retrieving all active clusters for a given AWS profile:
+osdctl org clusters --aws-profile my-aws-profile --aws-account-id 123456789
+`,
+		Args:          cobra.MaximumNArgs(1),
 		SilenceErrors: true,
 		Run: func(cmd *cobra.Command, args []string) {
-			cmdutil.CheckErr(SearchClusters(cmd, args))
+			orgId := ""
+			if len(args) > 0 {
+				orgId = args[0]
+			}
+
+			var status string
```

Comment:
```suggestion
			status := ""
```
to match 43 since these use the exact same logic, default to empty unless something is set specifically.